test/honeycomb-debug.test.ts

```diff
@@ -24,12 +24,14 @@ describe('when debug is set to true', () => {
       new HoneycombWebSDK({
         debug: true,
       });
-      expect(consoleSpy.mock.calls[1][0]).toContain(
-        'Honeycomb Web SDK Debug Mode Enabled',
+      expect(consoleSpy).toHaveBeenNthCalledWith(
```

Comment:
Did a lil refactor here to use Jest functions instead of array lookups because I found it really hard to pinpoint the right message quickly with the arrays 🙈

Reply:
Oh I love this!